docs/settings/enterprise-search-settings.asciidoc

```diff
@@ -0,0 +1,26 @@
+[role="xpack"]
+[[enterprise-search-settings-kb]]
+=== Enterprise Search settings in {kib}
+++++
+<titleabbrev>Enterprise Search settings</titleabbrev>
+++++
+
+You do not need to configure any settings to use Enterprise Search in {kib}. It is enabled by default.
```

Comment:
This statement is true on Elastic Cloud. For self-managed installations, it's necessary to configure `enterpriseSearch.host`.

Reply:
```suggestion
On Elastic Cloud, you do not need to configure any settings to use Enterprise Search in {kib}. It is enabled by default. On self-managed installations, you must configure `enterpriseSearch.host`.
```